config/helm/chart/default/values.yaml

```diff
@@ -56,6 +56,7 @@ webhook:
   limits:
     cpu: 300m
     memory: 128Mi
+  enableHighAvailability: false
```

Comment:
imo it would suffice if it was called `highAvailability` since the bool value already implies a enable / disable behaviour

Reply:
you are right, makes total sense. thank you